shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/Val.hs

```diff
@@ -47,7 +52,9 @@ class
     Show t,
     Typeable t,
     NFData t,
-    NoUnexpectedThunks t
+    NoUnexpectedThunks t,
+    ToCBOR t,
```

Comment:
We don't need these additional constraints here. There's nothing intrinsic in the notion of a `Val` that requires it to have `ToCBOR` instances - that's a serialisation concern. We should only add superclass constraints when they're actually required to declare a valid instance.

Reply:
without CBOR, the polymorphic TxOut can't have a CBOR instance (and adding the constraint there directly will propagate)

Reply:
OK. I think we will end up not needing them here, but leave them in for now and we can remove them later if possible.